services/worker-manager/src/provider.js

```diff
@@ -73,80 +41,45 @@ class Provider extends WMObject {
    * might wish to return {region: 'us-east-1'}
    */
   workerInfo({worker}) {
-    this._throw(errors.MethondUnimplemented, 'Provider.workerInfo()');
+    throw new Error('Method Unimplemented!');
   }
```

Comment:
It looks like these three methods are TBD?  I don't see `Provider.states` and I think we've discussed the "pull a list of workers" model before and don't like it.  So TBD is fine as long as it's planned to change :)